qualtran/bloqs/state_preparation/state_preparation_alias_sampling.py

```diff
@@ -123,27 +126,60 @@ def from_lcu_probs(
             alt=np.array(alt),
             keep=np.array(keep),
             mu=mu,
-            sum_of_lcu_coeffs=sum(lcu_probabilities),
+            sum_of_lcu_coeffs=sum(abs(x) for x in lcu_probabilities),
+        )
+
+    @classmethod
+    def from_n_coeff(
```

Comment:
This is purely for symbolics right? I think this should be documented in the docstring.

Reply:
Yup, I'll update the docstring